basis/bootstrap/image/primitives/primitives-docs.factor

```diff
@@ -19,7 +19,7 @@ HELP: primitive-quot
 { $description "Creates the defining quotation for the primitive. If 'vm-func' is a string, then it is prefixed with 'primitive_' and a quotation calling that C++ function is generated." } ;
 
 ARTICLE: "bootstrap.image.primitives" "Bootstrap primitives"
-"This vocab contains utilities for declaring primitives to be added to the bootstrap image. It is used by the file " { $snippet "resource:core/bootstrap/primitives.factor" }
+"This vocab contains utilities for declaring primitives to be added to the bootstrap image. It is used by " { $vocab-link "bootstrap.primitives" }
```

Comment:
This will fail unless we re-factor to make ``core/bootstrap/primitives.factor`` an actual vocabulary instead of a script.

Reply:
This was merged wrong, I meant to keep the `$snippet` line